app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,13 @@
   </head>
 
   <body>
+    <% if notice %>
```

Comment:
Do both of these ever happen at the same time? If not then `if/elsif` is probably better.